app/assets/stylesheets/application.scss

```diff
@@ -94,13 +86,15 @@ a {
 }
 
 a:link {
-color: #FF3300;
+  color: #FF3300;
 }
+
 a:visited {
-color: #CC0000;
+  color: #CC0000;
```

Comment:
Color literals like `#CC0000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#CC0000` should be written as `#cc0000`